scripts/setup.sh

```diff
@@ -322,6 +384,14 @@ if [ "$INSTALL" == true ]; then
 
 fi
 
+echo -e
+echo "- checking activated extensions in Dynatrace"
+get_activated_extensions_on_cluster "$DYNATRACE_URL" "$DYNATRACE_ACCESS_KEY"
+
+echo -e
+echo "- activating extensions in Dynatrace"
+activate_extension_on_cluster "$DYNATRACE_URL" "$DYNATRACE_ACCESS_KEY" "$EXTENSIONS_FROM_CLUSTER"
```

Comment:
This part should be after function repackage (changes in https://github.com/dynatrace-oss/dynatrace-gcp-function/pull/179)
- after downloading functions source (I need function dir to put configs there)